docs/_data/icons.json5

```diff
@@ -1,67 +1,67 @@
 [
-  'account',
-  'angle-bottom',
-  'angle-down',
-  'angle-end',
-  'angle-left',
-  'angle-right',
-  'angle-start',
-  'angle-top',
-  'angle-up',
-  'bell',
-  'billing',
-  'calendar',
-  'checkmark',
-  'checkmark-circle',
-  'cog',
-  'copy',
-  'download',
-  'envelope',
-  'exclamation-circle',
-  'exclamation-diamond',
-  'exclamation-triangle',
-  'export',
-  'external-link',
-  'filter',
-  'help-circle',
-  'info-circle',
-  'input-file',
-  'input-time',
-  'input-url',
-  'kbd-arrow-down',
-  'kbd-arrow-left',
-  'kbd-arrow-right',
-  'kbd-arrow-up',
-  'kbd-capslock',
-  'kbd-command',
-  'kbd-delete',
-  'kbd-eject',
-  'kbd-option',
-  'kbd-return',
-  'kbd-shift',
-  'kbd-space',
-  'kbd-tab',
-  'lock',
-  'minus',
-  'minus-circle',
-  'monitoring',
-  'payment',
-  'pencil',
-  'phone',
-  'plus',
-  'plus-or-minus',
-  'search',
-  'server',
-  'sort',
-  'sort-down',
-  'sort-up',
-  'support',
-  'tag',
-  'technical-change',
-  'technical-incident',
-  'ticketing',
-  'times',
-  'times-circle',
-  'trash',
-  'user',
+    { name: 'account', minver: '0.2.0' },
+    { name: 'angle-bottom', minver: '0.2.1' },
+    { name: 'angle-down' },
+    { name: 'angle-end', minver: '0.2.1' },
+    { name: 'angle-left' },
+    { name: 'angle-right' },
+    { name: 'angle-start', minver: '0.2.1' },
+    { name: 'angle-top', minver: '0.2.1' },
+    { name: 'angle-up' },
+    { name: 'bell', minver: '0.2.0' },
+    { name: 'billing', minver: '0.2.0' },
+    { name: 'calendar' },
+    { name: 'checkmark' },
+    { name: 'checkmark-circle', minver: '0.2.0' },
+    { name: 'cog' },
+    { name: 'copy', minver: '0.2.0' },
+    { name: 'download', minver: '0.2.0' },
+    { name: 'envelope' },
+    { name: 'exclamation-circle' },
+    { name: 'exclamation-diamond', minver: '0.2.1' },
+    { name: 'exclamation-triangle' },
+    { name: 'export', minver: '0.2.0' },
+    { name: 'external-link', minver: '0.2.0' },
+    { name: 'filter', minver: '0.2.0' },
+    { name: 'help-circle', minver: '0.2.0' },
+    { name: 'info-circle' },
+    { name: 'input-file' },
+    { name: 'input-time' },
+    { name: 'input-url' },
+    { name: 'kbd-arrow-down' },
+    { name: 'kbd-arrow-left' },
+    { name: 'kbd-arrow-right' },
+    { name: 'kbd-arrow-up' },
+    { name: 'kbd-capslock' },
```

Comment:
Many icons don't have a `minver` property, because they've been available since the beginning.